types/validator/index.d.ts

```diff
@@ -14,6 +14,7 @@
 //                 Matteo Nista <https://github.com/Mattewn99>
 //                 Roman Babiak <https://github.com/Almost-Infinity>
 //                 Daniel Freire <https://github.com/dcfreire>
+//                 Edilson Silva <https://github.com/edilson>
```

Comment:
If you're looking for credit, there's no need to add yourself here; the git history serves that function. This is the list of owners for the package which will trigger pings, code reviews, etc; are you wanting to do that or is this unintentional?

Reply:
Sorry @jakebailey it was unintentional, will remove that